.buildkite/pipeline.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - label: 'iohk-ops'
+    command: 'scripts/ci.sh'
+    env:
+      NIX_REMOTE: daemon
+      PATH: bin:/run/current-system/sw/bin
```

Comment:
Why is this needed? To find Nix?

Reply:
Yes, it wasn't finding `nix-shell`, since it was apparently clearing the environment,

Reply:
It's not naive, it ensure isolation.

We should try getting rid of `PATH` and use `/run/current-system/sw/bin/nix-shell -p nixStable bash coreutils --run "scripts/ci.sh"` for the command

Reply:
Agreed to impurely reference `nix-shell`, but don't extend `PATH`.